components/movementsensor/imuvectornav/imu.go

```diff
@@ -122,20 +114,23 @@ func newVectorNav(
 		return nil, err
 	}
 
-	speed := *newConf.Speed
-	if speed == 0 {
-		speed = 8000000
+	baudRate := *newConf.BaudRate
+	if baudRate == 0 {
+		baudRate = 9600
 	}
 
-	pfreq := *newConf.Pfreq
+	smp := 200
```

Comment:
Why is it called smp? Can we rename this variable to something that makes more sense?

Reply:
will rename to `pollFreq`